src/components/molecules/InputBox/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import AuthInput from "@/components/atoms/AuthInput";
+
+interface InputConfig {
+  type: string;
+  placeholder: string;
+  className: string;
+}
+
+interface Props {
+  inputs: InputConfig[];
+  onInputChange: (fieldName: string, value: string) => void; // Include onInputChange prop here
+}
+
+function InputBox({ inputs, onInputChange }: Props) {
+  return (
+    <div>
+      {inputs.map((input, index) => (
+        <AuthInput
+          key={`input-${index.toString()}`}
```

Comment:
템플릿 리터럴 사용하실때 숫자 뒤에 toString() 안 붙이셔도 됩니다!
```suggestion
          key={`input-${index}`}
```

Reply:
esLint 워닝이 떠서 저렇게 해 놓았습니다.

Reply:
아 그렇군요 그러면 현재 상태 유지해주세요 :)